api/integrations/lead_tracking/hubspot/lead_tracker.py

```diff
@@ -58,6 +58,12 @@ def create_lead(self, user: FFAdminUser, organisation: Organisation = None) -> N
 
         HubspotLead.objects.create(user=user, hubspot_id=response["id"])
 
+        if HubspotTracker.objects.filter(user=user).exists():
```

Comment:
nit: we can use walrus here and get rid of the exists

Reply:
Nice catch 👍🏼